src/examples/bluetooth-display.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
```

Comment:
github autocommented on a few things in this file.

Reply:
This file is still in flux until we have a finalized `parser` function. I've intentionally left a couple of things in here for the time being.